src/shared-components/immersiveModal/index.js

```diff
@@ -0,0 +1,82 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import OffClickWrapper from '../offClickWrapper';
+import CloseIcon from '../../svgs/icons/close-icon.svg';
+import {
+  ModalContainer,
+  Overlay,
+  CloseIconContainer,
+  CopyContainer,
+  Title,
+  Body,
+  Footer,
+} from './style';
+
+class ImmersiveModal extends React.Component {
+  static propTypes = {
+    children: PropTypes.node.isRequired,
+    canBeClosed: PropTypes.bool,
+    onClose: PropTypes.func,
+    header: PropTypes.node,
+  };
+
+  static defaultProps = {
+    canBeClosed: true,
+    onClose: () => {},
+  };
+
+  static Title = Title;
+
+  static Body = Body;
+
+  static Footer = Footer;
+
+  constructor(props) {
+    super(props);
+
+    this.htmlNode = document.querySelector('html');
+    this.domNode =
+      document.querySelector('#reactPortalSection') || document.body;
+  }
+
+  componentDidMount() {
+    this.htmlNode.classList.add('no-scroll');
```

Comment:
just wanted to check if `no-scroll` class was part of the inject global styles

Reply:
Confirmed -- just checked in radiance, gatsby, and main app and it's the same in all three.

Reply:
🙌